CHANGES.md

```diff
@@ -1,5 +1,5 @@
 # 2.8.0 (Unreleased)
-
+- [NEW] Added support for `/_search_disk_size` endpoint.
```

Comment:
Should we add a bit more info? e.g. `Added support for /_search_disk_size endpoint which retrieves disk size information for a specific search index.`